tasks/main.yml

```diff
@@ -26,6 +26,16 @@
   register: autoclean_output
   changed_when: "'Del' in autoclean_output.stdout"
 
+- name: APT | Check for cached .deb files
+  shell: "ls /var/cache/apt/archives/*.deb 2&> /dev/null | wc -l"
```

Comment:
This is only redirecting stderr to null. It seems to be working as desired for me in Vagrant.

* with `apt_clean` disabled nothing occurs
* with `apt_clean` enabled and deb files in the cache, they will be removed and only one task will be changed
* with `apt_clean` enabled and no deb files in the cache, nothing will happen